main.yml

```diff
@@ -716,6 +716,8 @@ service-registry:
     sub_apps:
       - id: instances
         title: "Service Registry Instances"
+      - id: service-registry
```

Comment:
It is no longer required to update nav in the `main.yml` file. It is not used to render any UI anymore.